### PR TITLE
document valuable resources to onboard developers

### PR DESCRIPTION
**Pull request by gregoryguillou**
This PR reference valuable documents to help people onboard on the project as developers. The goal is to help people onboard and to share ideas/resources.

### CODE REVIEW

**Review by internnos — COMMENTED** [review]
this is awesome

really helpful for onboarding!

great work

File: docs/developers.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+accounts. The reason is that you need go through an account to run transactions,
+including to create a contract from a class. That is why you should understand
+what is an account is and how to interact with it.
+
+The devil living in the details, there are specificities associated with
+account implementations. For instance, you might find that one account does not
+support the same signature as an other. To start with account, read:
+
+- [Starknet Account Abstraction Model - Part 1](https://community.starknet.io/t/starknet-account-abstraction-model-part-1/781)
+- [Starknet Account Abstraction Model - Part 1](https://community.starknet.io/t/starknet-account-abstraction-model-part-2/839)
```
Comment (0xLucqs) [inline docs/developers.md]: Starknet Account Abstraction Model - Part 2 instead of 1 I guess

File: docs/developers.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+- Openzeppelin Cairo contracts that includes an account
+  [implementation](https://github.com/OpenZeppelin/cairo-contracts/tree/main/src/openzeppelin/account)
+- The argent-x contract
+  [implementation](https://github.com/argentlabs/argent-contracts-starknet)
+
```
Comment (0xLucqs) [inline docs/developers.md:77]: Do we want a link to the account abstraction tutorial ? https://github.com/starknet-edu/starknet-accounts

Comment (gregoryguillou) [reply docs/developers.md:77]: Added! I did not know that resource 💯

File: docs/developers.md (hunk 1 above)
Comment (gregoryguillou) [reply docs/developers.md]: corrected. Thank you

**Review by 0xLucqs — APPROVED** [review]
lgtm

**Pull request merged**